.github/workflows/focus-ios-smoke

```diff
@@ -0,0 +1,124 @@
+name: "Focus/Firefox Tests"
+    
+on:
+    workflow_dispatch:
+
+env:
+    browser: focus-ios
+    xcode_version: 15.4
+    ios_version: 17.4
+    ios_simulator: iPhone 15
+    xcodebuild_test_plan: SmokeTest
+    xcodebuild_scheme: Focus
+    xcodebuild_target: XCUITest
+    test_results_directory: /Users/runner/tmp
+    
+jobs:
+    run-tests:
+        name: Build and run tests
+        runs-on: macos-14
```

Comment:
Currently, this workflow requires over 30 minutes to finish. If this workflow is run under the enterprise account, we may try with a `-large` runner. My fork is under my personal account so the `-large` runners are not available.